lib/internal/streams/operators.js

```diff
@@ -420,7 +421,7 @@ function take(number, options = undefined) {
 }
 
 module.exports.streamReturningOperators = {
-  asIndexedPairs,
+  asIndexedPairs: deprecate(asIndexedPairs, 'readable.asIndexedPairs is deprecated.', 'DEP0174'),
```

Comment:
@benjamingr this change causes a UT to fail as it checks for `Readable.prototype.asIndexedPairs.name`. Do we really care what the `.name` of this is?

Reply:
We do not, that was for spec compliance - as it's no longer part of the spec that check can be removed.